[2023-04-17 02:12:10,987] INFO Starting the log cleaner (kafka.log.LogCleaner)
[2023-04-17 02:12:11,067] INFO [kafka-log-cleaner-thread-0]: Starting (kafka.log.LogCleaner)
[2023-04-22 00:46:11,822] INFO Starting the log cleaner (kafka.log.LogCleaner)
[2023-04-22 00:46:11,875] INFO [kafka-log-cleaner-thread-0]: Starting (kafka.log.LogCleaner)
[2023-04-22 00:46:12,271] INFO Shutting down the log cleaner. (kafka.log.LogCleaner)
[2023-04-22 00:46:12,272] INFO [kafka-log-cleaner-thread-0]: Shutting down (kafka.log.LogCleaner)
[2023-04-22 00:46:12,273] INFO [kafka-log-cleaner-thread-0]: Stopped (kafka.log.LogCleaner)
[2023-04-22 00:46:12,273] INFO [kafka-log-cleaner-thread-0]: Shutdown completed (kafka.log.LogCleaner)
[2023-04-22 00:46:23,982] INFO Starting the log cleaner (kafka.log.LogCleaner)
[2023-04-22 00:46:24,036] INFO [kafka-log-cleaner-thread-0]: Starting (kafka.log.LogCleaner)
[2023-04-22 00:56:39,255] ERROR Failed to access checkpoint file cleaner-offset-checkpoint in dir /tmp/kafka-logs (kafka.log.LogCleaner)
org.apache.kafka.common.errors.KafkaStorageException: Error while reading checkpoint file /tmp/kafka-logs/cleaner-offset-checkpoint
Caused by: java.nio.file.NoSuchFileException: /tmp/kafka-logs/cleaner-offset-checkpoint
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:218)
	at java.base/java.nio.file.Files.newByteChannel(Files.java:380)
	at java.base/java.nio.file.Files.newByteChannel(Files.java:432)
	at java.base/java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:422)
	at java.base/java.nio.file.Files.newInputStream(Files.java:160)
	at java.base/java.nio.file.Files.newBufferedReader(Files.java:2921)
	at java.base/java.nio.file.Files.newBufferedReader(Files.java:2954)
	at org.apache.kafka.server.common.CheckpointFile.read(CheckpointFile.java:104)
	at kafka.server.checkpoints.CheckpointFileWithFailureHandler.read(CheckpointFileWithFailureHandler.scala:48)
	at kafka.server.checkpoints.OffsetCheckpointFile.read(OffsetCheckpointFile.scala:70)
	at kafka.log.LogCleanerManager.$anonfun$allCleanerCheckpoints$2(LogCleanerManager.scala:136)
	at scala.collection.Iterator$$anon$10.nextCur(Iterator.scala:594)
	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:608)
	at scala.collection.immutable.List.prependedAll(List.scala:152)
	at scala.collection.immutable.List$.from(List.scala:684)
	at scala.collection.immutable.List$.from(List.scala:681)
	at scala.collection.IterableFactory$Delegate.from(Factory.scala:288)
	at scala.collection.immutable.Iterable$.from(Iterable.scala:35)
	at scala.collection.immutable.Iterable$.from(Iterable.scala:32)
	at scala.collection.IterableFactory$Delegate.from(Factory.scala:288)
	at scala.collection.IterableOps.flatMap(Iterable.scala:684)
	at scala.collection.IterableOps.flatMap$(Iterable.scala:684)
	at scala.collection.AbstractIterable.flatMap(Iterable.scala:933)
	at kafka.log.LogCleanerManager.$anonfun$allCleanerCheckpoints$1(LogCleanerManager.scala:134)
	at kafka.log.LogCleanerManager.allCleanerCheckpoints(LogCleanerManager.scala:142)
	at kafka.log.LogCleanerManager.$anonfun$grabFilthiestCompactedLog$1(LogCleanerManager.scala:173)
	at kafka.log.LogCleanerManager.grabFilthiestCompactedLog(LogCleanerManager.scala:170)
	at kafka.log.LogCleaner$CleanerThread.cleanFilthiestLog(LogCleaner.scala:358)
	at kafka.log.LogCleaner$CleanerThread.tryCleanFilthiestLog(LogCleaner.scala:345)
	at kafka.log.LogCleaner$CleanerThread.doWork(LogCleaner.scala:332)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:96)
[2023-04-22 00:58:35,552] INFO Starting the log cleaner (kafka.log.LogCleaner)
[2023-04-22 00:58:35,613] INFO [kafka-log-cleaner-thread-0]: Starting (kafka.log.LogCleaner)
